core/src/test/java/novoda/lib/sqliteprovider/provider/SQLiteProviderLocalTest.java

```diff
@@ -212,10 +281,24 @@ private void insert(String uri, ContentValues cv) {
         provider.insert(Uri.parse("content://" + uri), cv);
     }
 
+    private void bulkInsert(String uri, ContentValues[] cv) {
+        provider.bulkInsert(Uri.parse("content://" + uri), cv);
+    }
+
     private void query(String uri) {
         provider.query(Uri.parse("content://" + uri), null, null, null, null);
     }
 
+    private ContentValues[] createContentValuesArray(int size) {
+        ContentValues[] bulkToInsert = new ContentValues[size];
+        ContentValues contentValues = new ContentValues();
+        contentValues.put("test", "test");
+        for (int i = 0; i < size; i++) {
+            bulkToInsert[i] = contentValues;
+        }
+        return bulkToInsert;
+    }
+
     public class SQLiteProviderImpl extends SQLiteContentProviderImpl {
```

Comment:
yeah you mean verify the methods are called rather than verify the implementation of our overridden method behaviour.  I assume this was here from the time before the dawn of mockito jar was available on Android